src/main/scala/dpla/ingestion3/messages/IngestMessage.scala

```diff
@@ -1,5 +1,13 @@
 package dpla.ingestion3.messages
 
+
+object IngestLogLevel {
+  val info: String = "info"
+  val warn: String = "warn"
+  val error: String = "error"
+  val fatal: String = "fatal"
+}
```

Comment:
Is the purpose of making these `String` types (as opposed to `Boolean` for example) that it makes logging output easier?

Reply:
Partially, but tbh I just never considered anything than String